ksp_plugin/interface_vessel.cpp

```diff
@@ -195,5 +195,29 @@ XYZ __cdecl principia__VesselVelocity(Plugin const* const plugin,
   return m.Return(ToXYZ(plugin->VesselVelocity(vessel_guid)));
 }
 
+void __cdecl principia__SetAngularMomentumConservation(
```

Comment:
Please create an issue to clean up these interfaces method as well as the static state in `PileUp`, listing all the stuff that needs to be removed.  Once we are satisfied with the fix, we want an easy way to remove the traces.

Reply:
#2535.